packages/seacas/libraries/ioss/src/main/CMakeLists.txt

```diff
@@ -251,48 +235,37 @@ else()
 	     )
 endif()
 
-if ( CGNS_CGNSLIST_BINARY )
+if ( CGNS_CGNSDIFF_BINARY )
 TRIBITS_ADD_ADVANCED_TEST(exodus64_to_unstructured_cgns
    TEST_0 EXEC io_shell ARGS ${DECOMP_ARG} --64 ${CMAKE_CURRENT_SOURCE_DIR}/test/8-block.g 8-block64.cgns
      NOEXEPREFIX NOEXESUFFIX
      NUM_MPI_PROCS ${NPROCS}
-   TEST_1 CMND ${CGNS_CGNSLIST_BINARY} ARGS -ld 8-block64.cgns
-     OUTPUT_FILE exodus64_to_unstructured_cgns.tmp
-   TEST_2 CMND grep ARGS -v Information exodus64_to_unstructured_cgns.tmp OUTPUT_FILE exodus64_to_unstructured_cgns.out
-   TEST_3 CMND diff ARGS ${CMAKE_CURRENT_SOURCE_DIR}/test/exodus_to_unstructured_cgns.gold exodus64_to_unstructured_cgns.out
+   TEST_1 CMND ${CGNS_CGNSDIFF_BINARY} ARGS -d 8-block64.cgns ${CMAKE_CURRENT_SOURCE_DIR}/test/8-block.cgns.gold
+     NUM_MPI_PROCS 1
```

Comment:
Is the following understanding correct?

1. `io_shell` runs as an MPI process and generates `8-block64.cgns`; the test fails before diffing if `io_shell` returns non-zero
2. After 1. is complete, test correctness is checked by diffing `8-block64.cgns` with `8-block.cgns.gold`

Reply:
Yes, that is correct understanding